<?xml version="1.0"?>
<launch>

    <node type="Battery" pkg="Battery" name = "battery_node" output="screen" required="true">
    </node>
 

    <node type="BaseNode" pkg="BaseNode" name = "base_node" output="screen" required="true">
    </node>

   

    <!-- <node type="joy_command" pkg="base_node" name = "joy_CMD">
    </node>
    <arg name="waypoint_goal_nameA" default="trajectory1"/>
    <arg name="waypoint_goal_nameB" default="trajectory2"/>
    <arg name="trajectory_goal_name" default="trajectory3"/>
    <arg name="mapless_mode" default="false"/>
    <arg name="vslam_mapping_mode" default="false"/>
    
    <arg name="device" default="/dev/input/js-logitech-wireless"/>

    <node type="joy_node" pkg="joy" name="joy">
        <param name="dev" value="$(arg device)"/>
        <param name="autorepeat_rate" value="10"/>
    </node>  -->
    
<!--
    <node type="joy_command.py" pkg="base_node" name="joy_command" output="screen">
        <param name="waypoint_goal_nameA" value="$(arg waypoint_goal_nameA)"/>
        <param name="waypoint_goal_nameB" value="$(arg waypoint_goal_nameB)"/>
        <param name="trajectory_goal_name" value="$(arg trajectory_goal_name)"/>
        
        <param name="mapless_mode" value="$(arg mapless_mode)"/>
    </node>
-->

    <!--
    <node type="keyboard_driver_node.py" pkg="xlib_keyboard_driver"
            name="keyboard_ros_driver" output="screen"/>
    -->

    <!--
    <node type="teleop_twist_keyboard.py" pkg="teleop_twist_keyboard" name="keyboard_input">
    </node>
    -->
    <!--
    <node type="teleop_node" pkg="teleop_twist_joy" name="joy_to_twist">
        <remap from="cmd_vel" to="joystick_set_velocity"/>
        
        <param name="scale_linear" value="0.5" unless="$(arg vslam_mapping_mode)"/>
        <param name="scale_linear" value="0.36" if="$(arg vslam_mapping_mode)"/>
        <param name="scale_angular" value="1.2"/> 
        <param name="enable_button" value="5"/>

        <param name="enable_turbo_button" value="4"/>
        <param name="scale_linear_turbo" value="0.9"/>
    </node>
    -->
</launch>
